src/keychain_fmt_plug.c

```diff
@@ -206,9 +206,8 @@ static int crypt_all(int *pcount, struct db_salt *salt)
 	int index = 0;
```

Comment:
Duplicate `index = 0`. Probably the same problem exists in many other of the patched `crypt_all()`, but just happened to be out of the patch context seen by default, thus I didn't notice it there.